Various edits

jpmckinney opened the pull request:
closes #1 

Readme

* Removed mapping guidance (should only be in OCDS for EU profile)
* Put EU-specific content in "Legal context" section
* Added example
* Added links

Codelists

* Added new codelists from latest eForms
* Remove empty Deprecated column
* Renamed criterionType.csv to awardCriterionType.csv and simplified descriptions compared to awardCriteria.csv

Schema

* Put Tender at top of patch
* Rename `awardWeightedCriteriaPerLot` to `awardCriteriaBreakdown` (to match budget breakdown extension)
* Remove `Weighted` word from definition names, as criteria are not necessarily weighted (e.g. order of importance is not a weight per 2014/24/EU)
* Rename `lotID` to `relatedLots` (to match lots extension)
* Remove repetitive prefixes: `awardCriteriaComplicated` -> `weightingDescription`, `awardCriteriaOrderJustification` -> `orderJustification`, `criterionType` -> `type`, `criterionName` -> `name`, `criterionDescription` -> `description`, `criterionNumber` -> `numbers`
* Add new fields from latest eForms
* Tidy titles and descriptions, including removing EU-specific references from descriptions
* Consistently use the property order: `title`, `description`, `type`
* Add `wholeListMerge` properties

I also removed the option to identify groups of lots. The lots extension doesn't require lot IDs and lot group IDs to be distinct. So, we'd need two properties to identify lots and lot groups, which becomes complicated. Instead, we can refer to the lots directly.

jpmckinney committed:
Various edits.

jpmckinney committed:
Fix documentationUrl with resolvable URI.

jpmckinney committed:
Add uniqueItems to array

jpmckinney committed:
Rename extension

jpmckinney committed:
Rename two codelists that can be used for selection critieria in eForms

ColinMaudry commented on README.md:
```diff
@@ -1,53 +1,56 @@
+          "1",
+          "2"
+        ],
+        "awardCriteria": [
```
Could you add an example with `fixed`? I don't understand how it's used.

jpmckinney replied:
Yes. Otherwise, the changes look good?

jpmckinney replied:
I added an example, based on https://www.felp.ac.uk/content/most-economically-advantageous-tender-meat and https://github.com/eprocurementontology/eprocurementontology/issues/157

jpmckinney committed:
Add .travis.yml

jpmckinney committed:
Add example for fixed price.

jpmckinney committed:
Put awardCriteria on Lot instead of by reference from Tender

jpmckinney committed:
Rename orderJustification to orderRationale

jpmckinney committed:
Add lots as dependency

jpmckinney commented:
As discussed, I moved `awardCriteriaBreakdown` from Tender to `awardCriteria` on Lot. Renamed the deep `awardCriteria` field to just `criteria`, since it is already in context of award criteria.

I also renamed `orderJustification` to `orderRationale` for consistency.
